stetho/src/test/java/com/facebook/stetho/inspector/database/DatabasePeerManagerTest.java

```diff
@@ -2,28 +2,35 @@
 
 package com.facebook.stetho.inspector.database;
 
+import java.io.File;
+import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
-
+import java.util.Set;
 import org.junit.Test;
 
 import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
 
 public class DatabasePeerManagerTest {
   @Test
   public void testTidyDatabaseList() {
-     String[] databases = {
-         "foo.db", "foo.db-journal",
-         "bar.db", "bar.db-journal", "bar.db-uid",
-         "baz.db", "baz.db-somethingelse",
-         "dangling.db-journal",
-     };
-     String[] expected = {
-         "foo.db",
-         "bar.db",
-         "baz.db", "baz.db-somethingelse",
-         "dangling.db-journal",
-     };
-     List<String> tidied = DatabasePeerManager.tidyDatabaseList(databases);
-     assertArrayEquals(expected, tidied.toArray());
+    Set<File> databases = new HashSet<File>();
```

Comment:
Changing it to a `List` should mean that the test can pretty much go back to what it was. Are you OK with the parameter on `tidyDatabaseList` changing from an array to a list?